doc/read-the-docs-site/quick-start.rst

```diff
@@ -0,0 +1,127 @@
+.. _quick_start:
+
+Quick Start
+=================================
+
+This guide gets you started on setting up the dev environment for a simple Cardano smart contract, with the on-chain validator written in Plutus Tx.
+There are several other options for writing on-chain validators, such as `Aiken <https://aiken-lang.org/>`_ and `OpShin <https://github.com/OpShin/opshin>`_, and you can refer to their respective documentation for how to use them.
+
+If you have any questions regarding this guide, open an issue in the `Plutus issue tracker <https://github.com/input-output-hk/plutus/issues>`_.
```

Comment:
It may be helpful to have this statement about questions and opening an issue in one location in the docs that applies across the board. Maybe on a "Contact us" page.

Reply:
Ok, I'll remove it.